Added SmartBASIC language syntax file for SmartBASIC language.

eriklins opened the pull request:
<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks,
depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request
and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass.
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can request a review from @packagecontrol-bot
to manually trigger an automated review
if you don't need to push a new commit.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"`
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.

You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists,
why you believe it is different and needed
below this line. -->

braver commented:
You might want to add the "language syntax" label. Also, we require a [semver](https://semver.org) format tag, e.g. `1.0.0`.  Package looks good otherwise 👍🏻

eriklins commented:
I added the label "language syntax" as requested and changed the tag 
from 1.0 to 1.0.0 to (hopefully) comply with the valid semver tag rules. 
It still gives an error during check. No idea what's still missing.


Am 30.03.2021 um 20:19 schrieb Koen Lageveen:
>
> You might want to add the "language syntax" label. Also, we require a 
> semver <https://semver.org> format tag, e.g. |1.0.0|. Package looks 
> good otherwise 👍🏻
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/wbond/package_control_channel/pull/8228#issuecomment-810475894>, 
> or unsubscribe 
> <https://github.com/notifications/unsubscribe-auth/AILUN7HWI47BDPZYAU7UQ7TTGII3BANCNFSM4ZRT7MWA>.
>

braver commented:
The review process seems to crash on something, but your package and the tag looks fine. 🤷🏻
